lib/experimental/Information/Headers/BaseHeader/index.tsx

```diff
@@ -0,0 +1,92 @@
+import { Button } from "@/components/Actions/Button"
+import { IconType } from "@/components/Utilities/Icon"
+import {
+  AvatarVariant,
+  renderAvatar,
+} from "@/experimental/Information/Avatars/utils"
+import { cn } from "@/lib/utils"
+
+interface BaseHeaderProps {
+  title: string
+  avatar?: AvatarVariant
+  description?: string
+  eyebrow?: React.ReactNode
+  footer?: React.ReactNode
+  primaryAction?: {
+    label: string
+    icon?: IconType
+    onClick: () => void
+  }
+  secondaryActions?: Array<{
+    label: string
+    icon?: IconType
+    onClick: () => void
+    variant?: "outline" | "critical"
+  }>
```

Comment:
Could we perhaps get a separate exported type for this? Same for primary action. It might come in handy :)